doc/USER_GUIDE.md

```diff
@@ -22,7 +22,7 @@ Users have the following options for specifying their own values:
 | Variable | Default value | Description |
 | - | - | - |
 | `additional_yum_repos` | `""` |  |
-| `ami_component_description` | ```(k8s: {{ user `kubernetes_version` }}, docker: {{ user `docker_version` }}, containerd: {{ user `containerd_version` }})``` |  |
+| `ami_component_description` | ```(k8s: {{ user `kubernetes_version` }}, containerd: {{ user `containerd_version` }})``` |  |
```

Comment:
How this get updated for 1.25 and below versions?

Reply:
using a pin to 1.28 for now, so that following https://github.com/awslabs/amazon-eks-ami/pull/1546#discussion_r1440703896 at least stably produces one table per variant

Reply:
I think omitting the defaults make sense. We can just add a note that the default depends on the Kubernetes version, and can call out notable k8s version boundaries  in the `Description` column where it matters.